AEPMessaging/Sources/Feed.swift

```diff
@@ -0,0 +1,27 @@
+/*
+ Copyright 2023 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPServices
+import Foundation
+
+@objc(AEPFeed)
+@objcMembers
+public class Feed: NSObject, Codable {
+    /// Identification for this feed, represented by the AJO Surface URI used to retrieve it
+    public var id: String
+    
+    /// Friendly name for the feed, provided in the AJO UI
+    public var name: String
```

Comment:
I'm assuming we shall extract this from the feed item meta info, if provided. Rethinking if we really see a need to expose it here?

Reply:
i suppose it may be useful for presentation purposes in the UI.